orbitx/flight_gui.py

```diff
@@ -458,38 +470,43 @@ def _draw_sphere(self, planet):
                 v0=self._vpython.vertex(pos=self._vpython.vector(0, 0, 0)),
                 v1=self._vpython.vertex(pos=self._vpython.vector(-5, 0, 30)),
                 v2=self._vpython.vertex(pos=self._vpython.vector(-5, 0, -30)))
+
             obj = self._vpython.compound([body, head, wing, wing2])
             obj.texture = self._vpython.textures.metal
             obj.pos = self._posn(planet)
             obj.axis=self._ang_pos(planet.heading)
             obj.radius=planet.r/2
-            #obj.make_trail=True
-            #obj.retain=100
-            #obj.shininess=0.1
+            obj.shininess=0.1
             obj.length = planet.r * 2
 
 
-            #obj = self._vpython.cone(
-            #    pos=self._posn(planet),
-            #    axis=self._ang_pos(planet.heading),
-            #    radius=planet.r / 2,
-            #    make_trail=False,
-            #    retain=100,
-            #    shininess=0.1
-            #)
-            #obj.length = planet.r
+            """
+            obj = self._vpython.cone(
+                pos=self._posn(planet),
+                axis=self._ang_pos(planet.heading),
+                radius=planet.r / 2,
+                make_trail=False,
+                retain=100,
+                shininess=0.1
+            )
+            obj.length = planet.r
+            """
 
             obj.arrow = self._unit_velocity(planet)
             self._habitat = planet
             self._vpython.attach_arrow(obj, 'arrow')#scale=planet.r * 1.5)
+            self._habitat_trail = self._vpython.attach_trail(obj, retain=100)
```

Comment:
is there a reason we need this attach_trail functionality instead of setting make_trail?

Reply:
I tried make_trail but this function doesn't seem to be working with compound object.